### DISCUSSION TITLE
[v10][client][typescript] No completion in client

### DISCUSSION THREAD

**Discussion by angelhdzmultimedia** [wobsoriano/trpc-nuxt#109] [Q&A]
I'm following this guide https://trpc-nuxt.vercel.app/get-started/installation

Added:

```ts
 build: {
    transpile: ['trpc-nuxt']
  },
```

Versions:

@trpc/client: 10.35.0
@trpc/server: 10.35.0
trpc-nuxt: 0.10.6

### Issue

No completion in the client:

![image](https://github.com/wobsoriano/trpc-nuxt/assets/10191085/f0ed13b4-85ef-41ea-b2d1-0ea9c41271d7)

### Attempts to solve it

1. Added `trpc-nuxt` to modules array in `nuxt.config.ts`
2. Added `typescript: {strict: true}`.
3. Reloaded VSCode
4. Reinstalled all dependencies

**Comment by angelhdzmultimedia** [comment]
### Update

`$client` is also unknown:

![image](https://github.com/wobsoriano/trpc-nuxt/assets/10191085/639438bb-d9d2-462e-9d8c-cd1e54856017)

**Comment by wobsoriano** [comment]
Did you run `npm run dev` yet?

I just tried on a fresh app

<img width="687" alt="Screenshot 2023-07-20 at 10 15 55 PM" src="https://github.com/wobsoriano/trpc-nuxt/assets/13049130/c9446dd2-7b4c-4d55-9eab-1667372677af">

**Comment by angelhdzmultimedia** [comment]
Wow. It was a silly mistake on my end, after a while of debugging.
The Nuxt client plugin was under a trpc directory and client.ts file, so it didn't get installed.

Working now! Thank you a lot for your time and for this module and my apologies.

![image](https://github.com/wobsoriano/trpc-nuxt/assets/10191085/252f3f0e-afa4-489d-9409-f0ba04e360d8)

Wishing you health and peace.

**Reply by wobsoriano** [reply]
Thank you as well :)

**Reply by angelhdzmultimedia** [reply]
When creating the files for the server, I wrote trpc/[file here] many times, so muscular memory kicked in when creating the client plugin! Oops! 😁🤦‍♂️🤝💚
